x-pack/plugins/security_solution/cypress/e2e/detection_rules/alerts_table_threat_indicator.cy.ts

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { CustomRule } from '../../objects/rule';
+import { getNewThreatIndicatorRule } from '../../objects/rule';
+import {
+  ALERTS_COUNT,
+  ALERT_TABLE_ADDITIONAL_CONTROLS,
+  ALERT_TABLE_SHOW_THREAT_INDICATOR,
+} from '../../screens/alerts';
+import { goToRuleDetails } from '../../tasks/alerts_detection_rules';
+import { cleanKibana } from '../../tasks/common';
+import {
+  createAndEnableRule,
+  fillAboutRuleAndContinue,
+  fillDefineIndicatorMatchRuleAndContinue,
+  fillScheduleRuleAndContinue,
+  selectIndicatorMatchType,
+  waitForAlertsToPopulate,
+} from '../../tasks/create_new_rule';
+import { esArchiverLoad, esArchiverUnload } from '../../tasks/es_archiver';
+import { login, visit, visitWithoutDateRange } from '../../tasks/login';
+import { ALERTS_URL, RULE_CREATION } from '../../urls/navigation';
+
+describe('Threat Indicator', () => {
+  before(() => {
+    cleanKibana();
+    esArchiverLoad('threat_indicator');
+    esArchiverLoad('suspicious_source_event');
+    login();
+    const rule = getNewThreatIndicatorRule();
+    visitWithoutDateRange(RULE_CREATION);
+    selectIndicatorMatchType();
+    fillDefineIndicatorMatchRuleAndContinue(rule);
+    fillAboutRuleAndContinue({ name: rule.name, description: rule.description } as CustomRule);
+    fillScheduleRuleAndContinue(rule);
+    createAndEnableRule();
```

Comment:
failures in `before` hook are not retried https://docs.cypress.io/guides/guides/test-retries#How-It-Works
Given that all these commands interact with UI, it highly probable it might be a single failure, that will stop executing the whole test case.

Instead, I would suggest to use [API call to create threatIndicator rule](https://github.com/elastic/kibana/blob/8.6/x-pack/plugins/security_solution/cypress/tasks/api_calls/rules.ts#L211)

And put it in `beforeEach` hook.
Thus
1. It will be retriable
2. it would take significantly less time to create rule using API, instead of using UI. And would also be less flaky

Reply:
Here is also an [example](https://github.com/elastic/kibana/blob/8.6/x-pack/plugins/security_solution/cypress/e2e/detection_rules/bulk_edit_rules.cy.ts#L130-L149), how multiple rule types can be created by using API calls